e2e-tests/pages/insurance/account/suspended/emailSent.js

```diff
@@ -1,7 +1,9 @@
 const emailSentPage = {
   weSentLinkTo: () => cy.get('[data-cy="we-sent-link-to"]'),
   checkYourEmail: () => cy.get('[data-cy="check-your-email"]'),
-  havingProblemsHeading: () => cy.get('[data-cy="having-problems-heading"]'),
+  emailPrefix: () => cy.get('[data-cy="email-prefix"]'),
+  emailLink: () => cy.get('[data-cy="email-link"]'),
```

Comment:
since being used twice, could these be shared fields?

Reply:
Good shout - moved to `pages/shared` selectors 👍